_author_instructions/01_policies.md

```diff
@@ -321,10 +325,10 @@ Domain Dedication (which can be found at
 https://creativecommons.org/publicdomain/zero/1.0/)."
 [See the analysis of this language here](https://theunitedstates.io/licensing/).
 
-## Prior Publication
+## Prior Publication and Plagarism
 
 Documents should not have been submitted in the current form to another journal, or be simultaneously under consideration for publication another journal.
-Preprints are not considered prior publication.
+Importatly, preprints are not considered prior publication.
```

Comment:
"Importantly"; but is this word needed here? I think it's important only if it's relevant to the person and is unlikely to need emphasis.